packages/ds-healthcare-gov/src/components/Header/Menu.tsx

```diff
@@ -1,10 +1,29 @@
 /* eslint-disable jsx-a11y/no-redundant-roles */
 import MenuLinks from './MenuLinks';
-import PropTypes from 'prop-types';
 import React from 'react';
 import classnames from 'classnames';
 
-const Menu = function (props) {
+interface MenuProps {
+  links?: {
+    identifier?: string;
+    href: string;
+    label: React.ReactNode;
+    onClick?: (...args: any[]) => any;
+  }[];
```

Comment:
There's probably a little bit of reuse you could get here from the Links interface in Header, but you'd also have to extend it, so might not be worth